File: 06Adapter/Students/2019/SebastianPiekarek/src/ZAD1/Seacraft.java
package ZAD1;

public class Seacraft implements ISeacraft{

    int speed;


    public Seacraft(){
        this.speed = 0;
    }

    @Override
    public void IncreaseRevs() {
        speed += 10;
        System.out.println("Seacraft engine increases revs to " + speed + " knots");
    }

    public int getSpeed() {
        return speed;
    }
}

File: 06Adapter/Students/2019/PokrzywniakRobert/AdapterPattern/AdapterPattern/Zad1/Seacraft.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AdapterPattern.Zad1
{
 public class Seacraft : ISeacraft {
        int speed = 0;

        public virtual void IncreaseRevs() {
            speed += 10;
            Console.WriteLine("Seacraft engine increases revs to " + speed + " knots");
        }
        public int Speed
        {
            get { return speed; }
        }
    }
}
